src/firewall/client.py

```diff
@@ -3034,6 +3034,9 @@ def addIPSet(self, name, settings):
 
     @handle_exceptions
     def getZoneNames(self):
+        return self.raw_getZoneNames()
+
+    def raw_getZoneNames(self):
```

Comment:
These are only used for `show_state_and_exit()`. If one fails I would expect them all to fail. So in `show_state_and_exit()` couldn't we have a single failure scenario that prints "unknown" for all things. i.e. fetch all the days in `try`, and if you get an exception set all the printed info to "unknown".